src/systemtest/resources/LocalRepoSystemTest/testRelativePathing/parent1_test-repo_master/authorship.json

```diff
@@ -1 +1 @@
t":"fake"},{"lineNumber":3,"author":{"gitId":"fakeAuthor"},"content":"fake"},{"lineNumber":4,"author":{"gitId":"fakeAuthor"},"content":"fake"},{"lineNumber":5,"author":{"gitId":"harryggg"},"content":"//@@author harryggg"},{"lineNumber":6,"author":{"gitId":"harryggg"},"content":"line 1"},{"lineNumber":7,"author":{"gitId":"harryggg"},"content":"line 2"},{"lineNumber":8,"author":{"gitId":"harryggg"},"content":"line 3"},{"lineNumber":9,"author":{"gitId":"harryggg"},"content":"//@@author"},{"lineNumber":10,"author":{"gitId":"fakeAuthor"},"content":"fake"},{"lineNumber":11,"author":{"gitId":"fakeAuthor"},"content":"fake"},{"lineNumber":12,"author":{"gitId":"-invalidGitUsername_TreatedAsUnknownUser"},"content":"//@@author -invalidGitUsername_TreatedAsUnknownUser"},{"lineNumber":13,"author":{"gitId":"-invalidGitUsername_TreatedAsUnknownUser"},"content":"unknown"},{"lineNumber":14,"author":{"gitId":"-invalidGitUsername_TreatedAsUnknownUser"},"content":"System.out.println(\"//@@author invalidAuthorLineFormat\"); unknown"},{"lineNumber":15,"author":{"gitId":"-invalidGitUsername_TreatedAsUnknownUser"},"content":"unknown"},{"lineNumber":16,"author":{"gitId":"-invalidGitUsername_TreatedAsUnknownUser"},"content":"//@@author"},{"lineNumber":17,"author":{"gitId":"fakeAuthor"},"content":"fake"},{"lineNumber":18,"author":{"gitId":"harryggg invalidAuthorLineFormat"},"content":"//@@author harryggg invalidAuthorLineFormat"},{"lineNumber":19,"author":{"gitId":"harryggg invalidAuthorLineFormat"},"content":"fake"},{"lineNumber":20,"author":{"gitId":"harryggg invalidAuthorLineFormat"},"content":"//@@author"},{"lineNumber":21,"author":{"gitId":"fakeAuthor"},"content":"unknown"},{"lineNumber":22,"author":{"gitId":"fakeAuthor"},"content":"unknown"}],"authorContributionMap":{"fakeAuthor":9,"-invalidGitUsername_TreatedAsUnknownUser":5,"harryggg":5,"harryggg invalidAuthorLineFormat":3}},{"path":"blameTest.java","fileType":"java","lines":[{"lineNumber":1,"author":{"gitId":"harryggg"},"content":"line 1"},{"lineNumber":2,"author":{"gitId":"harryggg"},"content":"line 2"},{"lineNumber":3,"author":{"gitId":"fakeAuthor"},"content":"fake"},{"lineNumber":4,"author":{"gitId":"harryggg"},"content":"line 3"}],"authorContributionMap":{"fakeAuthor":1,"harryggg":3}},{"path":"newFile.java","fileType":"java","lines":[{"lineNumber":1,"author":{"gitId":"harryggg"},"content":"line 1"},{"lineNumber":2,"author":{"gitId":"harryggg"},"content":"line 2"}],"authorContributionMap":{"harryggg":2}},{"path":"newPos/movedFile.java","fileType":"java","lines":[{"lineNumber":1,"author":{"gitId":"harryggg"},"content":"line 1"},{"lineNumber":2,"author":{"gitId":"harryggg"},"content":"line 2"},{"lineNumber":3,"author":{"gitId":"harryggg"},"content":"line 3"},{"lineNumber":4,"author":{"gitId":"harryggg"},"content":"line 4"}],"authorContributionMap":{"harryggg":4}},{"path":"space test.txt","fileType":"txt","lines":[{"lineNumber":1,"author":{"gitId":"chan-j-d"},"content":"1"}],"authorContributionMap":{"chan-j-d":1}}]
```

Comment:
Will you need to modify these JSON files as well after merging the changes in https://github.com/reposense/testrepo-Alpha/pull/14

Reply:
Yes my apologies for missing that! Will update it

Reply:
Actually, after a second thought, I think this may become too tedious for a single change of author name ...... You may revert back to the original -invalidGitUsername_TreatedAsUnknownUser and close https://github.com/reposense/testrepo-Alpha/pull/14 accordingly. I will approve the pull request after that. Sorry for the inconvenience

Reply:
Ah, no worries! Can I check then, should I still continue with making the names better in a separate PR?

Reply:
I do not think that is necessary